### DISCUSSION TITLE
Displaying specific validation control

### DISCUSSION THREAD

**Discussion by benounnas** [fomantic/Fomantic-UI#655] [Behaviors (Api, Form Validation, Visibility)] [answered]
hello, 
i'm kind new using Fomantic UI, i want to show only one exactly rule of specific validation form
for example

`$('form')
  .form({
  
    fields: {
      empty: {
        identifier  : 'empty',
        rules: [
          
          {
            type   : 'minLength[100]',
            prompt : 'rule length'
          }, 
          {
            type   : 'empty',
            prompt : 'rule empty'
          }
        ]
      }
    }
  })
;`

i want only to show empty not the result of the 2nd valdation 
thank you  
https://jsfiddle.net/zrd84wgb/7/

**Comment by ko2in** [comment]
Your fiddle is broken, and not properly created. Adjust your fiddle with only front-end code, so, someone can test and help you.

**Comment by lubber-de** [comment, marked as answer]
Regardless of the provided fiddle it is currently not implemented to only show the first occuring invalid rule, but always all of them when submitting the form. 
The only current option to accomplish the desired functionality would be by testing the fields manually via behaviors `is valid(fieldname)` and `add prompt(identifier, errors)`
https://fomantic-ui.com/behaviors/form.html#/settings

**Comment by benounnas** [comment]
i'm sorry, i messed up the fiddle, thought i uploaded the clean code, i'll check with `is valide(fieldname)`

**Comment by benounnas** [comment]
well basically the solution i was searching for is this 
https://jsfiddle.net/2kyrj87s/
thank you @ColinFrick
